From 956747b76a2d8c0cdb26430560b751066569a076 Mon Sep 17 00:00:00 2001
From: Tobias <tobiasmicheler@outlook.de>
Date: Fri, 26 Apr 2024 14:41:54 +0200
Subject: [PATCH] Appstream: Fix appstream errors for Flatpak

Fixing the following errors when Flathub
is building Wireshark:
desktop-app-launchable-missing
appstream-summary-too-long
---
 resources/freedesktop/org.wireshark.Wireshark.metainfo.xml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/resources/freedesktop/org.wireshark.Wireshark.metainfo.xml b/resources/freedesktop/org.wireshark.Wireshark.metainfo.xml
index 9827d0751e8..6748684d621 100644
--- a/resources/freedesktop/org.wireshark.Wireshark.metainfo.xml
+++ b/resources/freedesktop/org.wireshark.Wireshark.metainfo.xml
@@ -7,8 +7,9 @@
 
 <component type="desktop">
     <id>org.wireshark.Wireshark</id>
+    <launchable type="desktop-id">org.wireshark.Wireshark.desktop</launchable>
     <name>Wireshark</name>
-    <summary>Wireshark is the world's foremost protocol analyzer</summary>
+    <summary>Network protocol analyzer</summary>
     <developer_name>The Wireshark developer community</developer_name>
     <content_rating type="oars-1.1"/>
     <metadata_license>CC0-1.0</metadata_license>
-- 
GitLab

